### ISSUE TITLE
Unreasonable project structure

### ISSUE THREAD

**Issue by WAY29** [sensepost/dwn#7]
i think plans directory  should not be placed outside the dwn folder, this will cause site-packages to be contaminated (for example, if there is another project named plans)

**Comment by leonjza** [comment]
Yeah, I actually plan on moving plans to their own repo at some stage and have them updatable without requiring a version bump in dwn.
